lms/models/application_instance.py

```diff
@@ -142,11 +142,7 @@ def update_lms_data(self, params: dict):
         ):
             # If we already have a LMS guid linked to the AI
             # and we found a different one report it to sentry
-            LOG.exception(
```

Comment:
Probably better to have a custom exception here but for this PoC this is enough

Reply:
@jon-betts somehow I marked this as resolved myself :shrug: . Yeah, changing it to a new exception type